cosmology.c

```diff
@@ -33,17 +33,11 @@ void init_cosmology(Cosmology * CP_in)
                   / (3*C*C*C*HUBBLE*HUBBLE)
                   / (CP->HubbleParam*CP->HubbleParam);
 
-    /* Neutrino + antineutrino background temperature as a ratio to T_CMB0
-     * Note there is a slight correction from 4/11
-     * due to the neutrinos being slightly coupled at e+- annihilation.
-     * See Mangano et al 2005 (hep-ph/0506164)
-     * The correction is (3.046/3)^(1/4), for N_eff = 3.046 */
-    double TNu0_TCMB0 = pow(4/11., 1/3.) * 1.00328;
-
-    /* For massless neutrinos,
-     * rho_nu/rho_g = 7/8 (T_nu/T_cmb)^4 *N_eff,
-     * but we absorbed N_eff into T_nu above. */
-    CP->OmegaNu0 = CP->OmegaG * 7. / 8 * pow(TNu0_TCMB0, 4) * 3;
+    init_omega_nu(&CP->ONu, CP->MNu, TimeBegin, CP->HubbleParam, CP->CMBTemperature);
+    /* Neutrinos will be included in Omega0, if massive.
```

Comment:
This convention is inconvenient. I remember redshift dependency of massive neutrino is not a**3 -- due to the distribution function. So If you include Oncdm in Omega0, then special care has to be taken for the redshift dependency...

Reply:
We may want to drastically change the input parameters such that it takes a proper set (CLASS set) of parameters instead of the crippled Omega0.